src/components/CreateVideo.js

```diff
@@ -100,19 +167,23 @@ const CreateVideo = (props) => {
       artist: results[songItem].artist.name,
       deezer_id: results[songItem].id,
       video_title: videoTitle.title,
+      im_group: videoParams.im_group,
+      jitter: videoParams.jitter,
+      depth: videoParams.depth,
+      truncation: videoParams.truncation,
+      pitch_sensitivity: videoParams.pitch_sensitivity,
+      tempo_sensitivity: videoParams.tempo_sensitivity,
+      smooth_factor: videoParams.smooth_factor,
     });
+
     setSongLoading(false);
   };
 
   const submitForm = (event) => {
     event.preventDefault();
     // Need to create postVideo action in redux for this to work
-    props.postVideo(
-      localStorage.getItem("token"),
-      selectedSong,
-      videoParams,
-      props.history
-    );
+    console.log(selectedSong);
+    props.postVideo(localStorage.getItem("token"), selectedSong, props.history);
```

Comment:
Hahaha, more formatting differences between my computer and yours.  Also, finishing that videoParams POST issue.